pkg/store/networkconfiguration.go

```diff
@@ -33,51 +31,23 @@ type NetworkConfiguration struct {
 	ConfigurationChanges map[ConfigName]change.ID
 }
 
-// CreateNetworkStoreWithName creates a NetworkConfiguration object
-func CreateNetworkStoreWithName(name string, user string) (*NetworkConfiguration, error) {
+// CreateNetworkStore creates a NetworkConfiguration object
+// If name is left blank then a name will be generated by a hash algorithm
+func CreateNetworkStore(name string, user string, changes map[ConfigName]change.ID) (*NetworkConfiguration, error) {
```

Comment:
I'm thinking that it might be best to rename this to NetworkConfigurationStore

Reply:
Good catch - It's not creating the store either - I've changed it to CreateNetworkConfiguration